### DISCUSSION TITLE
allow for datasets queries based on brain regions, and number of neurons recorded?

### DISCUSSION THREAD

**Discussion by duutabib** [dandi/helpdesk#113] [Ideas]
I am interested in datasets of neural populations (~10K neurons) from different brain regions. However, I am unable to query at that resolution using your tools, or am I missing something?

Thanks kindly in advance.

**Comment by CodyCBakerPhD** [comment]
> I am interested in datasets of neural populations (~10K neurons) from different brain regions.

If you mean simultaneous recordings of that many neurons+regions, your best best would be to search for 'whole brain microscopy' which has a couple of datasets on that scale

[Glia Accumulate Evidence that Actions Are Futile and Suppress Unsuccessful Behavior](https://dandiarchive.org/dandiset/000350): Number of ROIs (neurons and/or glia) on the order of hundreds of thousands, but very low temporal resolution

[Thermoregulatory Responses Midbrain](https://dandiarchive.org/dandiset/000236) + [Hindbrain](https://dandiarchive.org/dandiset/000237): similar in style/scale to the glia one, but each dataset is specific to the subregions


If on the other hand, you're OK with the cells not being detected simultaneously but instead being observed in different sessions with different subjects (different regions each time) then you should definitely check these two out

[IBL - Brain Wide Map](https://dandiarchive.org/dandiset/000409)

[Mesoscale Activity Map Dataset](https://dandiarchive.org/dandiset/000363/0.230613.1608)



> However, I am unable to query at that resolution using your tools, or am I missing something?

Improved search UI is something we're actively working on - what kind of search tool would you envision for helping discover datasets like this?

**Comment by duutabib** [comment]
Hello Cody,

Many thanks for your quick response. I will have a look into these. I am
actually interested in simultaneous recordings of ROIs from different
regions ideally in spontaneous activity states.

I envision a UI that allows one to query along these lines individually and
simultaneously. For example, filter based on number of ROIs, state of the
brain, region of the brain etc. I'd imagine such datasets are already
available but, the absence of standardization of descriptors might make it
difficult for one to find these datasets.

I will like to thank you again, and will let you know there are any
challenges as I plow along.


Best wishes,
*Duut.Abib*
*------------------------------------------------------------------------------------------------------*

**Comment by duutabib** [comment]
Hello Cody,

For IBL datasets, I am having a challenge accessing `passivePeriods` for
some datasets, any chance you could help me with that?


Best wishes,
*Duut.Abib*
*------------------------------------------------------------------------------------------------------*

**Comment by CodyCBakerPhD** [comment]
The [IBL passive data release](https://int-brain-lab.github.io/iblenv/notebooks_external/loading_passive_data.html#Loading-Passive-Data) is still in progress as far as I know, so it hasn't made its way over to DANDI yet

Last correspondence I had with them on the matter noted a new release by the end of the year

**Comment by duutabib** [comment]
I will brute force it then. Many thanks Cody. I will be in touch if I have
any other questions.

Going back to comments on the UI:
 https://knowledge.brain-map.org/data?limit=25&offset=0&sort=species.name~ASC
<https://knowledge.brain-map.org/data?limit=25&offset=0&sort=species.name~ASC>
have
something along the lines I was thinking, although it doesn't account for
brain states (example: spontaneous activity).  I hope this makes
things clearer.

Do have a nice day!

Best wishes,
*Duut.Abib*
*------------------------------------------------------------------------------------------------------*
